crypto/rand/drbg_lib.c

```diff
@@ -377,7 +379,9 @@ int RAND_DRBG_reseed(RAND_DRBG *drbg,
     drbg->state = DRBG_ERROR;
     if (drbg->get_entropy != NULL)
         entropylen = drbg->get_entropy(drbg, &entropy, drbg->strength,
-                                   drbg->min_entropylen, drbg->max_entropylen);
+                                      drbg->min_entropylen,
+                                      drbg->max_entropylen,
+                                      prediction_resistance);
```

Comment:
My indentation was even worse, but yours is still off-by-one, you need to add one more space (lines 382-384).